plan/new_physical_plan_builder.go

```diff
@@ -346,8 +347,8 @@ func (p *DataSource) tryToGetMemTask(prop *requiredProp) (task taskProfile, err
 		TableAsName: p.TableAsName,
 	}.init(p.allocator, p.ctx)
 	memTable.SetSchema(p.schema)
-	rb := &rangeBuilder{sc: p.ctx.GetSessionVars().StmtCtx}
-	memTable.Ranges = rb.buildTableRanges(fullRange)
+	rb := &ranger.Builder{Sc: p.ctx.GetSessionVars().StmtCtx}
```

Comment:
Could you define a helper method to get StmtCtx, I saw redundant "p.ctx.GetSessionVars().StmtCtx"